src/looprestoration_tmpl_16.rs

```diff
@@ -1092,27 +1076,27 @@ unsafe extern "C" fn loop_restoration_dsp_init_arm(
 
     cfg_if! {
         if #[cfg(target_arch = "aarch64")] {
-            (*c).wiener[0] = Some(dav1d_wiener_filter7_16bpc_neon);
-            (*c).wiener[1] = Some(dav1d_wiener_filter5_16bpc_neon);
+            (*c).wiener[0] = LoopRestorationFilterFn::Wiener7_Neon;
+            (*c).wiener[1] = LoopRestorationFilterFn::Wiener5_Neon;
         } else {
             // TODO(perl): enable assembly routines here
-            // (*c).wiener[0] = Some(dav1d_wiener_filter_neon);
-            // (*c).wiener[1] = Some(dav1d_wiener_filter_neon);
+            // (*c).wiener[0] = LoopRestorationFilterFn::Bpc16(dav1d_wiener_filter_neon);
+            // (*c).wiener[1] = LoopRestorationFilterFn::Bpc16(dav1d_wiener_filter_neon);
```

Comment:
It would be nice to get these arm asm functions ported first to properly test all the changes here.  cc @thedataking.

Reply:
Did you forget to update the `::Bpc16`s?